cpp/src/c_api/capi_helper.hpp

```diff
@@ -36,6 +36,10 @@ void sort_by_key(raft::handle_t const& handle,
                  raft::device_span<key_t> keys,
                  raft::device_span<value_t> values);
 
+template <typename T>
```

Comment:
I would suggest moving this to `include/cugraph/detail`.  Perhaps a new file called `collect_comm_wrapper.hpp` within that directory to mirror the name of the functions we're wrapping.  We could ultimately wrap other methods from the `collect_comm.cuh` file as well.

Reply:
resolved